exec/plugin-commands-script-runners/src/exec.ts

```diff
@@ -120,6 +127,19 @@ export async function handler (
       }
     }
   }
+
+  if (opts.resumeFrom) {
+    const resumeFromPackagePrefix = Object.keys(opts.selectedProjectsGraph)
+      .find((prefix) => opts.selectedProjectsGraph?.[prefix]?.package.manifest.name === opts.resumeFrom)
```

Comment:
if `opts.selectedProjectsGraph` is not defined, I think an error should be thrown

Reply:
I'm not quite sure about this situation, can you help explain why it is not defined? thanks ~

I looked at the type definition and it doesn't seem to be `undefined`?
<img width="813" alt="image" src="https://user-images.githubusercontent.com/41503212/210121160-44da2f5f-14d4-44d6-9935-b09e3a820748.png">

Reply:
but you do check if it is defined by using optional chaining (`opts.selectedProjectsGraph?`)

Instead, you can check if it exists before line 132

Reply:
Hmmm, I'm using the optional chain based on the usage in line 159.

I wonder if we can remove the optional chaining if it really won't be `undefined`, I don't see other examples of checking `opts.selectedProjectsGraph ` elsewhere, like in `run` or `runRecursive`, Please let me know if there is something missing. Thanks a lot